ci/.env

```diff
@@ -6,5 +6,5 @@
 export CURRENT_CI='travis'
 export MULTI_BRANCH=0
 export REPOSITORY='api-platform/demo'
-export DEPLOYMENT_BRANCH='master'
+export DEPLOYMENT_BRANCH='feature/blackfire'
```

Comment:
```suggestion
export DEPLOYMENT_BRANCH='master'
```

Reply:
This is in order to allow travis to deploy testing branch during dev.
I'll reset it to master when ready to merge.